README.md

```diff
@@ -67,6 +67,23 @@ Plug 'nvim-treesitter/nvim-treesitter'
 Plug 'cuducos/yaml.nvim'
 ```
 
+## Winbar
+
+```lua
+
+-- set autocmd to get yaml path in winbar
+
+vim.api.nvim_create_autocmd({ "FileType" }, {
+	pattern = { "yaml" },
+	callback = function()
+  	vim.opt_local.winbar = "%{%v:lua.require'yaml_nvim'.get()%}"
+	end,
+})
+
```

Comment:
This code should match the plugin style (also, we don't need that comment; if you still want to say what this block does, this is already a text file, we can add a paragraph between the heading and code-block).

```suggestion
vim.api.nvim_create_autocmd({ "FileType" }, {
	pattern = { "yaml" },
	callback = function()
		vim.opt_local.winbar = [[%{%v:lua.require("yaml_nvim").get()%}]]
	end,
})
```